src/components/application_manager/test/CMakeLists.txt

```diff
@@ -38,34 +38,43 @@ include_directories(
   ${GMOCK_INCLUDE_DIRECTORY}
 )
 
+#=============== COMMANDS =====================
 set(COMMANDS_PATHS
   ${CMAKE_CURRENT_SOURCE_DIR}/commands
   ${CMAKE_CURRENT_SOURCE_DIR}/commands/hmi/hmi_notifications
```

Comment:
@AGritsevich seems like this line is redundant. Root path `${CMAKE_CURRENT_SOURCE_DIR}/commands` is already included